### ISSUE TITLE
[gql_build] Build data for Mutations

### ISSUE THREAD

**Issue by smkhalsa** [gql-dart/gql#40]
As you can see [here](https://github.com/gql-dart/gql/blob/master/gql_code_builder/lib/src/operation/data.dart#L35), we are currently only building data classes for `Query`s, not `Mutation`s.

**Comment by klavs** [comment]
True. It should be quite simple to fix by mapping operation type to correct object name. The only thing to remember is that `{ fooBar }` is a valid query and then the operation type (I think) is `null`.

We can ignore for now the fact that there might be a custom schema definition:
```
schema {
  query: OtherQuery
  mutation: SomeMutation
  subscription: SuperScription
}
```

**Comment by smkhalsa** [comment]
> `{ fooBar }` is a valid query and then the operation type (I think) is `null`

Actually, shouldn't the operation type in that case be `query`?

**Comment by smkhalsa** [comment]
closed in #41
